CONTRIBUTING.md

```
@@ -18,48 +18,50 @@ Guidelines
 
-* Create a new feature branch from master. <br/> The new branch must follow the name convention
-`pybandits-XXXX feature_name` where `XXXX`= number of the issue linked to this branch and `feature_name`= short
-description of the new feature (e.g. add predict function to sMAB)
+* Create a new feature branch from `develop`, which is the default branch and the base of every pull request.
+<br/> The new branch must follow the name convention `feature/<short_description>` where `<short_description>`=
+short description of the new feature (e.g. `feature/add_predict_function_to_smab`)
 
 * Checkout on the new branch and write the code of the new feature.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-$ git checkout feature/pybandits-XXXX feature_name
+$ git checkout develop
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)